cmd/crc/cmd/delete.go

```diff
@@ -37,12 +37,14 @@ func runDelete(arguments []string) {
 	if clearCache {
 		deleteCache()
 	}
-
-	_, err := machine.Delete(deleteConfig)
-	if err != nil {
-		errors.Exit(1)
+	yes := input.PromptUserForYesOrNo("Do you want to delete the crc VM, it will cause data lose in the VM", globalForce)
```

Comment:
I'd go with `Do you want to delete the crc VM (you'll lose all data in the VM)` in brackets, as `PromptUserForYesOrNo()` would add a question mark at the end of the sentence.